client/src/battle/player_form.rs

```diff
@@ -1,11 +1,16 @@
-use super::{BattleCallback, PlayerOverridables};
-use framework::prelude::Texture;
+use super::{
+    Artifact, BattleCallback, BattleSimulation, Entity, Living, Player, PlayerOverridables,
+};
+use crate::bindable::EntityId;
+use crate::resources::Globals;
+use framework::prelude::{Texture, Vec2};
 use std::sync::Arc;
 
 #[derive(Default, Clone)]
 pub struct PlayerForm {
     pub activated: bool,
     pub deactivated: bool,
+    pub deactivate_on_weakness: bool,
```

Comment:
\-

Marking lines that should be removed with a `-`, starting with this line

Otherwise looks good to merge